brainlit/algorithms/image_processing.py

```diff
@@ -0,0 +1,96 @@
+def Bresenham3D(x1, y1, z1, x2, y2, z2):
```

Comment:
in order to add a function to the package, you need to write tests. Please see e.g. [here](https://github.com/neurodata/brainlit/blob/develop/tests/test_spline_fxns.py)